troposphere/static/js/components/login/LoginSplash.jsx

```diff
@@ -26,7 +23,9 @@ export default React.createClass({
                         <h2 className="t-title option__title">
                           {"Log in with your " + window.ORG_NAME + " Account"}
                         </h2><hr/>
-                        <p className="option__description">{"Login to browse Atmosphere's list of available images and select one to launch a new instance."}</p>
+                        <p className="option__description">
+                            {"Login to browse Atmosphere's list of available images and select one to launch a new instance."}
```

Comment:
We _can_, but I think the `{" ... "}` are using to avoid the "unmatched" diff rendering of GitHub when there is a single apostrophe.